eslint.config.mjs

```
@@ -5,17 +5,23 @@ import eslintConfigPrettier from "eslint-config-prettier";
 import sonarjs from "eslint-plugin-sonarjs";
 import importPlugin from "eslint-plugin-import";
+import vuePlugin from "eslint-plugin-vue";
+import vueParser from "vue-eslint-parser";
 
 export default [
   {
-    files: ["{src,test}/**/*.{js,ts,yaml,yml}", "assets/html/js/**/*.js"],
+    files: [
+      "{src,test}/**/*.{js,ts,yaml,yml,vue}",
+      "assets/html/js/**/*.js",
+    ],
```

ESLint

[error] Replace `⏎······"{src,test}/**/*.{js,ts,yaml,yml,vue}",⏎······"assets/html/js/**/*.js",⏎····` with `"{src,test}/**/*.{js,ts,yaml,yml,vue}",·"assets/html/js/**/*.js"`

(prettier/prettier)